backend-config/namespace_config.go

```diff
@@ -80,10 +86,18 @@ func (nc *namespaceConfig) getFromAPI(ctx context.Context) (map[string]ConfigT,
 	if nc.namespace == "" {
 		return config, fmt.Errorf("namespace is not configured")
 	}
+	useUpdateAfter := !(nc.lastUpdatedAt.IsZero()) && nc.useIncrementalConfigUpdates
```

Comment:
No need to call `IsZero()` if `nc.useIncrementalConfigUpdates` is `false`:

```suggestion
	useUpdateAfter := nc.useIncrementalConfigUpdates && !nc.lastUpdatedAt.IsZero()
```